### PR TITLE
Add ListCardComponent and MapCardComponent with Associated Styles and Data for the BEAT plan. 

### PR DESCRIPTION
**Pull request by KarthikShetty818**
###  Add ListCardComponent and MapCardComponent with Associated Styles and Data for the BEAT plan. 

- Added 'UserImg.png' to load on each card if the user does not have a profile image.
- Added ListCardComponent to render items using FlatList in MyBeatPlan.screen.tsx.
- Included MapCardComponent to render horizontally scrollable items using FlatList in MyBeatPlan.screen.tsx.
- Updated dummyData.ts to include sample data for beatListCardData.
- Imported NavIconImg.svg, phoneGreenImg.svg, UserDefaultImg.svg, and beatProfileImg.png assets.
- Created ListCardComponent and associated style file listCardComponent.style.ts.
- Implemented ListCardComponent to display card details including name, address, distance, time, and image.
- Created MapCardComponent and associated style file mapCardComponent.style.ts.
- Implemented MapCardComponent to display card details including count, name, address, distance, time, and navigation link.
- Ensured all components are responsive and styled consistently with theme colors.

### CODE REVIEW

File: app/screens/beat/components/listCardComponent/listCardComponent.style.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import { COLORS } from '@/theme/colors';
+import { heightToRatio, widthToRatio } from '@/utils/commonMethods';
+import {StyleSheet} from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    padding: 10,
+    marginVertical: 5,
+    backgroundColor: '#ffffff',
```
Comment (kartikDeloitte) [inline app/screens/beat/components/listCardComponent/listCardComponent.style.ts]: use from COLORS

File: app/screens/beat/components/listCardComponent/listCardComponent.style.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  image: {
+    width: 32,
+    height: 32,
+  },
+  infoTextContainer: {
+    width: widthToRatio(215),
+    padding: 10,
+    marginHorizontal: 5,
+  },
+  nameText: {
```
Comment (kartikDeloitte) [inline app/screens/beat/components/listCardComponent/listCardComponent.style.ts]: use variant and text from react native paper

File: app/screens/beat/components/listCardComponent/listCardComponent.style.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    fontSize: 14,
+    fontWeight: 'bold',
+    color: COLORS.black,
+  },
+  addressText: {
+    fontSize: 12,
+    color: '#796767',
```
Comment (kartikDeloitte) [inline app/screens/beat/components/listCardComponent/listCardComponent.style.ts]: use from COLORS

File: app/screens/beat/components/listCardComponent/listCardComponent.style.ts [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  navImg: {
+    marginHorizontal: 5,
+    position: 'absolute',
+    marginVertical: 5,
+  },
+  distanceTimeText: {
+    marginLeft: 18,
+    padding: 5,
+    fontSize: 12,
+    color: '#656767',
```
Comment (kartikDeloitte) [inline app/screens/beat/components/listCardComponent/listCardComponent.style.ts]: COLORS

File: app/screens/beat/components/listCardComponent/listCardComponent.tsx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    image: string;
+    name: string;
+    address: string;
+    distance: string;
+    time: string;
+  }
+
+function ListCardComponent({ image, name, address, distance, time }: ListCardComponentProps) {
+  return (
+        <View style={styles.container}>
```
Comment (kartikDeloitte) [inline app/screens/beat/components/listCardComponent/listCardComponent.tsx]: use card component already present in components

File: app/screens/beat/components/mapCardComponent/mapCardComponent.style.ts [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+import { green } from 'react-native-reanimated/lib/typescript/Colors';
+
+const styles = StyleSheet.create({
+  container: {
+    width: widthToRatio(285),
+    padding: 10,
+    paddingTop: 5,
+    marginVertical: 10,
+    marginHorizontal: 5,
+    backgroundColor: '#ffffff',
```
Comment (kartikDeloitte) [inline app/screens/beat/components/mapCardComponent/mapCardComponent.style.ts]: remove COLORS from every where

File: app/screens/beat/components/mapCardComponent/mapCardComponent.tsx [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+    count: number;
+    name: string;
+    address: string;
+    distance: string;
+    time: string;
+  }
+
+function MapCardComponent({ count, name, address, distance, time }: MapCardComponentProps) {
+  return (
+        <View style={styles.container}>
```
Comment (kartikDeloitte) [inline app/screens/beat/components/mapCardComponent/mapCardComponent.tsx]: use card component from already made components

File: app/utils/dummyData.ts [hunk 8]
```diff
@@ -404,3 +404,211 @@ export const orderDashboard = {
     ],
   },
 };
+
```
Comment (kartikDeloitte) [inline app/utils/dummyData.ts:407]: why have you added this code?

**Comment by kartikDeloitte** [comment]
create a component folder in beatPlan and the component there.

**Comment by KarthikShetty818** [comment]
### [1deb926] Implemented the changes 

- Update color codes to use values from COLORS file
- Wrap card component with card wrapper
- Apply text variants

**Comment by KarthikShetty818** [comment]
### [5a0ae69] Create CommonCard Component for ListCard and MapCard Components.

•	Created CommonCard component to encapsulate shared structure and styles between ListCard and MapCard components.
•	Ensured adherence to DRY (Don't Repeat Yourself) principles by moving common code to a single reusable component.
•	Updated ListCard and MapCard components to utilize the new CommonCard component.
•	Added UserImg.png as the default profile image for users without a profile picture.
•	Implemented ListCard component to render items using FlatList in MyBeatPlan.screen.tsx.
•	Included MapCard component to render horizontally scrollable items using FlatList in MyBeatPlan.screen.tsx.
•	Updated dummyData.ts to include sample data for beatListCardData.
•	Imported necessary assets: NavIconImg.svg, phoneGreenImg.svg, UserDefaultImg.svg, and beatProfileImg.png.
•	Ensured all components are responsive and styled consistently with theme COLORS.
•	Deleted 2 folders ‘listCardComponent’ and ‘mapCardComponent’ in the ‘app/screens/beat/components’ as they were not required.

**Pull request merged**